### PR TITLE
Introduce expandPageSize API request parameter

### PR DESCRIPTION
**Pull request by vulkodava**
expandPageSize - allows specifying page size for direct and not direct related resources

### CODE REVIEW

**Comment by ivaylomitrev** [comment]
Unrelated to any files - I don't think `Map<String, String>` belongs to the commit description. It's an implementation detail :)

File: v1/examples.md [hunk 1]
```diff
@@ -431,6 +436,7 @@ Content-type: application/json
         ]
     },
     "expand": ["parties"],
+    "expandPageSize": {"parties": 12}
```
Comment (ivaylomitrev) [inline v1/examples.md]: Missing comma after the line

File: v1/model.md [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
 - `expand`
+- `expandPageSize`
 - `attributes`
 - `sort`
  
 You can jump right to the section you are interested in:
 - [Model diagram](#model-diagram)
 - [Data](#data)
 - [Expand](#expand)
+- [Expand Page Size](#expandPageSize)
```
Comment (ivaylomitrev) [inline v1/model.md]: The link has to be `#expand-page-size` to work as a link.

File: v1/model.md [hunk 3]
```diff
@@ -100,6 +112,8 @@ The table columns are as follows:
 
 The `expand` sections describe the supported values for the `expand` field in requests.
 
+The `expandPageSize` can be set to any and only value that is set in the `expand` list of the particular request.
```
Comment (ivaylomitrev) [inline v1/model.md]: I think `and only` is not needed.

**Review by ivaylomitrev — CHANGES_REQUESTED** [review]
Unrelated to any files - I don't think `Map<String, String>` belongs to the commit description. It's an implementation detail :)

**Comment by phillipj** [comment]
Looks pretty good to me, excited to see this in practise 🕺🏻

File: v1/examples.md (hunk 1 above)
Comment (vulkodava) [reply v1/examples.md]: Fix introduced with 41f8be6

File: v1/model.md (hunk 2 above)
Comment (vulkodava) [reply v1/model.md]: Fix introduced with 41f8be6

File: v1/model.md (hunk 3 above)
Comment (vulkodava) [reply v1/model.md]: Change introduced with 41f8be6

**Comment by vulkodava** [comment]
> expandPageSize - Map<String, Integer> - allows specifying page size
> for direct and not direct related resources

The commit message was changed to not include the implementational detail.

**Comment by vulkodava** [comment]
> Unrelated to any files - I don't think `Map<String, String>` belongs to the commit description. It's an implementation detail :)

The commit message was changed to not include the implementation detail.

**Review by ivaylomitrev — APPROVED** [review]

**Pull request merged**